libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/types.ts

```diff
@@ -68,10 +68,10 @@ export type Docs = {
   BirthCertificate: ReactNativeFile | undefined;
   PhotoId: ReactNativeFile | undefined;
   SocialSecurityCard: ReactNativeFile | undefined;
-  ConsentForm: ReactNativeFile[] | undefined;
-  HmisForm: ReactNativeFile[] | undefined;
-  IncomeForm: ReactNativeFile[] | undefined;
-  OtherClientDocument: ReactNativeFile[] | undefined;
+  ConsentForm?: ReactNativeFile[] | undefined;
```

Comment:
Any particular reason for including `undefined` explicitly? 
My understanding was the `?` already does that.